.editorconfig

```diff
@@ -0,0 +1,7 @@
+[*.rs]
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
Is this really required? It's not very useful to be _in_ the repository. Shall I rebase and remove this commit?